src/components/GraphLayout.vue

```diff
@@ -59,21 +33,26 @@
             <circle cx="50" cy="50" r="50" class="link-start" />
           </marker>
         </defs>
-        <path
-          v-for="(link, index) in links"
-          :key="index"
-          class="link stroke-gray-700 dark:stroke-gray-100"
-          :class="{ active: activeLinks.includes(link) }"
-        >
-          <title>{{ link.label }}</title>
-        </path>
+        <g class="link-container" v-for="(link, index) in links" :key="index">
+          <path
+            class="link stroke-gray-700 dark:stroke-gray-100"
```

Comment:
This is not related to this arrow alignement commit, 
But I think this component should be not dependent of Tailwind, so one does not force the host application to use it.